server.js

```diff
@@ -301,20 +301,27 @@ app.get('/questionform', function(req, res) {
         return res.redirect('/');
     }
 
-    var html = questionForm();
-
-    return res.status(200).send(html);
-});
-
-app.get('/re', function(req, res){
-    var html = regexForm();
-    res.status(200).send(html);
+    return res.status(200).render('question-creation',{
+        questionType: common.questionTypes
+    });
 });
 
-app.get('/mc', function(req, res){
-    var html = multipleChoiceForm();
-    res.status(200).send(html);
-});
+app.post('/formanswer', function(req, res){
+    switch (req.body.qType){
+        case common.questionTypes.REGULAR.value:
+            res.status(200).render(
+                common.questionTypes.REGULAR.template,{
```

Comment:
can we have that `{` on the next line? so it would be like 
``` javascript
common.questionTypes.REGULAR.template,
{answerForm:true});
```
same applies to the section below